flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableITCase.java

```diff
@@ -852,6 +852,7 @@ public void testStartFromGroupOffsetsNone() {
                         + " (2, 6),\n"
                         + " (2, 7),\n"
                         + " (2, 8)\n";
+        Thread.sleep(2000);
```

Comment:
Why does this sleep 2000 millseconds? I don't think this is the fundamental solution to the problem.

Reply:
fix it.